CMakeLists.txt

```diff
@@ -65,11 +65,24 @@ if(NOT PARAVIEW_ENABLE_PYTHON)
     "Tomviz requires PARAVIEW_ENABLE_PYTHON to be enabled. "
     "Please rebuild ParaView with PARAVIEW_ENABLE_PYTHON set to TRUE.")
 endif()
-if(NOT PARAVIEW_QT_VERSION STREQUAL "5")
-  message(FATAL_ERROR
-    "Tomviz requires PARAVIEW_QT_VERSION to be 5, please rebuild ParaView.")
+
+load_cache(${ParaView_DIR}
```

Comment:
This only works in the superbuild by accident since we use ParaView's build tree there. An install tree will be…well, it doesn't have `CMakeCache.txt`. [This MR](https://gitlab.kitware.com/vtk/vtk/merge_requests/5373) should be used in the future.

Reply:
Our past builds of tomviz have also relied on `load_cache`. I believe the build instructions require linking to a build tree, not an install tree. But we can look into using the variable instead when that gets merged.

Reply:
Yes, we haven't worried about building against an installed ParaView. I think there are some other concerns if we need to do that, such as the use of third party headers/libraries too.

Reply:
> <using build tree>

This is likely because on macOS, ParaView didn't install a useful SDK. It now does.

> I think there are some other concerns if we need to do that, such as the use of third party headers/libraries too.

VTK/ParaView's third party libraries no longer conflict with anything (headers, libraries, or symbols), so that concern should be resolved.